tests/page/pageTest.ts

```diff
@@ -27,4 +27,4 @@ export type PageTestFixtures = {
   isElectron: boolean;
 };
 
-export const test = baseTest.declare<PageTestFixtures>();
+export const test = baseTest.declare<PageTestFixtures, { browser: Browser }>();
```

Comment:
Page tests do not provide a browser instance, for example when running with Android.

Reply:
(need to be moved outside of the page folder)